glidertools/optics.py

```diff
@@ -126,12 +126,12 @@ def par_dark_count(par, dives, depth, time):
     return par_dark
 
 
-def backscatter_dark_count(bbp, depth):
+def backscatter_dark_count(bbp, depth, percentile=95):
     """
     Calculates an in situ dark count from the backscatter sensor.
 
     The in situ dark count for the backscatter sensor is calculated from the
-    95th percentile between 200 and 400m.
+    95th percentile between 200 and 400m. Change this threshold by adjusting percentile = ...
```

Comment:
Is this potentially confusing? Since you can adjust the percentile, I would not say `95th` in here.